de-lib/src/main/java/org/iplantc/de/theme/base/client/collaborators/GroupDisplayStrings.properties

```diff
@@ -1,3 +1,5 @@
+addGroup=Add List
+deleteGroup=Delete List
 deleteGroupConfirm=By deleting the collaborator list {0}, any data that has been shared with this list will immediately be un-shared.  Do you wish to continue deleting the collaborator list?
```

Comment:
Maybe @dennisroberts71 can better comment, but in my mind deleting a list is similar to deleting a user (actually deleting a user, not just removing them from your Collaborators).  It can't still be shared with them if they don't exist.
But you are correct - if you remove a user from your Collaborators, the sharing persists.  If you remove/delete a list, the sharing does not persist.